kestest/gateway_test.go

```diff
@@ -633,3 +626,28 @@ func mustDecodeB64(s string) []byte {
 	}
 	return b
 }
+
+func clean(ctx context.Context, client *kes.Client, t *testing.T) {
+	iter, err := client.ListKeys(ctx, "*")
+	if err != nil {
+		t.Fatalf("Cleanup: failed to list keys: %v", err)
+	}
+	defer iter.Close()
+
+	names := []string{}
+	keysInfo, err := iter.Values(-1)
```

Comment:
`iter.Values(-1)` already collects all `KeyInfo` objects into one slice. You don't need to clone this slice again. 
You can simply remove:
```
for _, keyInfo := range keysInfo {
	names = append(names, keyInfo.Name)
}
```

An just do one for loop:
```
for _, info := range keyInfos {
		if err = client.DeleteKey(ctx, info.Name); err != nil {
			t.Errorf("Cleanup: failed to delete '%s': %v", name, err)
		}
	}
```

Reply:
Done